src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -66,6 +66,23 @@ function computeEmbeddedMaxWidth(tagName, contentWidth) {
     return contentWidth;
 }
 
+/**
+ * Check if there is an ancestor node with name 'comment'.
+ * Finding node with name 'comment' flags that we are rendering a comment.
+ * @param {TNode} tnode
+ * @returns {Boolean}
+ */
+function isInsideComment(tnode) {
+    let currentNode = tnode;
```

Comment:
What does tnode stand for here? Would you mind renaming this to make that clear?

Reply:
Oh, just saw that we have `TNodeRenderer` down there. Nvm